lib/guardian.ex

```diff
@@ -83,7 +83,7 @@ defmodule Guardian do
   * `claims` - Any custom claims that you want to use in your token
   * `opts` - Options for the token module and callbacks
 
-  For more information on options see the documentation for your Token Module.
+  For more information on options see the documentation for your TokenModule.
```

Comment:
I don't believe this shouldn't be referring to any naming in your code.

People will try to find `TokenModule` in their code for this change for sure and then open issues because they can't find the module.

Reply:
This change isn't necessary.  We're not referencing a specific `TokenModule` rather a `Token Module` that they will create.  Probably doesn't need to be capitalized?